Error running cognigy CLI 1.6.0 - missing dependency fs-extra

When running the version 1.6.0 of the cognigy CLI, the following error is returned:

```
PS D:\> cognigy
node:internal/modules/cjs/loader:1252
  throw err;
  ^

Error: Cannot find module 'fs-extra'
Require stack:
- C:\ProgramData\nvm\v22.11.0\node_modules\@cognigy\cognigy-cli\build\lib\aiagents.js
- C:\ProgramData\nvm\v22.11.0\node_modules\@cognigy\cognigy-cli\build\commands\clone.js
- C:\ProgramData\nvm\v22.11.0\node_modules\@cognigy\cognigy-cli\build\program\program.js
- C:\ProgramData\nvm\v22.11.0\node_modules\@cognigy\cognigy-cli\build\program\index.js
- C:\ProgramData\nvm\v22.11.0\node_modules\@cognigy\cognigy-cli\build\cognigy.js
    at Function._resolveFilename (node:internal/modules/cjs/loader:1249:15)
    at Function._load (node:internal/modules/cjs/loader:1075:27)
    at TracingChannel.traceSync (node:diagnostics_channel:315:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:218:24)
    at Module.require (node:internal/modules/cjs/loader:1340:12)
    at require (node:internal/modules/helpers:141:16)
    at Object.<anonymous> (C:\ProgramData\nvm\v22.11.0\node_modules\@cognigy\cognigy-cli\build\lib\aiagents.js:30:25)
    at Module._compile (node:internal/modules/cjs/loader:1546:14)
    at Object..js (node:internal/modules/cjs/loader:1689:10)
    at Module.load (node:internal/modules/cjs/loader:1318:32) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'C:\\ProgramData\\nvm\\v22.11.0\\node_modules\\@cognigy\\cognigy-cli\\build\\lib\\aiagents.js',
    'C:\\ProgramData\\nvm\\v22.11.0\\node_modules\\@cognigy\\cognigy-cli\\build\\commands\\clone.js',
    'C:\\ProgramData\\nvm\\v22.11.0\\node_modules\\@cognigy\\cognigy-cli\\build\\program\\program.js',
    'C:\\ProgramData\\nvm\\v22.11.0\\node_modules\\@cognigy\\cognigy-cli\\build\\program\\index.js',
    'C:\\ProgramData\\nvm\\v22.11.0\\node_modules\\@cognigy\\cognigy-cli\\build\\cognigy.js'
  ]
}
```

The NPM package `fs-extra` seems to be missing as a dependency in the package.json file. 

The CLI works correctly with version 1.5.5

I am using node 22.11.0 via nvm on a Windows 11 machine

Same here. I've created a Dockerfile for the CLI, such that you don't have to have node and other stuff installed on your PC to use it:

```Dockerfile
FROM docker.io/library/node:23.2.0-alpine3.20

ENV NODE_ENV production
ENV NODE_OPTIONS "--no-deprecation"

WORKDIR /usr/src/app
RUN npm install -g @cognigy/cognigy-cli@v1.6.3

ENTRYPOINT ["cognigy"]
```

Building it is succesful, but when you try to use it, you get `Error: Cannot find module 'fs-extra'`

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@cognigy/cognigy-cli",
-  "version": "1.6.0",
+  "version": "1.6.4",
   "description": "Cognigy Command Line Interface",
   "main": "./build/cognigy.js",
   "scripts": {
@@ -61,6 +61,7 @@
     "epub2": "3.0.2",
     "express": "4.21.2",
     "form-data": "3.0.0",
+    "fs-extra": "^11.3.0",
     "gpt-3-encoder": "1.1.4",
     "html-to-text": "9.0.5",
     "inquirer": "7.3.3",

